sdk/core/azure-core/src/main/java/com/azure/core/util/paging/ContinuablePagedFlux.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util.paging;
+
+import reactor.core.publisher.Flux;
+
+/**
+ * A contract that represents a Flux that provides the ability to operate on pages of type
+ * {@link ContinuablePage} and individual items in such pages. This type supports user-provided
+ * continuation tokens, allowing for restarting from a previously-retrieved continuation token.
+ *
+ * @param <C> the type of continuation token
+ * @param <T> the type of elements in the page
+ * @param <P> the type of page
+ *
+ * @see Flux
+ * @see ContinuablePage
+ */
+public abstract class ContinuablePagedFlux<C, T, P extends ContinuablePage<C, T>> extends Flux<T> {
```

Comment:
Since all methods in this class are `abstract` change this to an interface instead.

Reply:
It's an abstract class because it needs to extend from reactor Flux class.